magento/adobe-stock-integration
# #1688: Saved image from Adobe Stock is selected/highlighted, but the "Add Selected" button is not displayed

### Description (*)

This PR fixes issue related to #1688

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1688: Saved image from Adobe Stock is selected/highlighted, but the "Add Selected" button is not displayed

### Manual testing scenarios (*)

1. Go to **Catalog - Categories**, 
2. Expand **Content**, click **Select from Gallery**
3. Click **Search Adobe Stock**
4. Save an Image Preview
5. Observe if the Image is selected/highlighted and if **Add Selected** button appeared

### Expected result (*)
The Image is selected/highlighted and **Add Selected** button appeared
![selected_er](https://user-images.githubusercontent.com/45624059/89038332-d30a6000-d348-11ea-9907-23a43abf9e77.png)
